common/components/org.wso2.carbon.analytics.util/pom.xml

```diff
@@ -84,13 +76,14 @@
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Bundle-Name>${project.artifactId}</Bundle-Name>
-                        <Private-Package>org.wso2.carbon.analytics.common.internal.*</Private-Package>
+                        <Private-Package>org.wso2.carbon.analytics.common.spark.internal.*</Private-Package>
                         <Export-Package>
-                            !org.wso2.carbon.analytics.common.internal.*,
-                            org.wso2.carbon.analytics.common.spark.*
+                            !org.wso2.carbon.analytics.common.spark.internal.*,
+                            org.wso2.carbon.analytics.common.spark.udf*
                         </Export-Package>
                         <Import-Package>
-                            org.wso2.carbon.analytics.spark.core.*
+                            org.wso2.carbon.analytics.spark.core.*,
```

Comment:
Hi Niranjan,

Thank you for pointing this and I have added these version range in following PR : https://github.com/wso2/shared-analytics/pull/131 

Thanks,
SajithD